libs/core/src/lib/product-switch/product-switch/product-switch.component.html

```diff
@@ -14,9 +14,10 @@
             <button
                 fd-button
                 [attr.aria-label]="ariaLabel"
+                [attr.title]="ariaLabel"
                 class="fd-shellbar__button fd-product-switch__control"
-                [glyph]="'grid'"
-                [fdType]="'transparent'"
+                glyph="grid"
```

Comment:
Maybe we should also add this to guideline too. So people are not mix expression that angular needs to evaluate this simple form. 

And also if. there will be always this, maybe we should think to use `@Attribute`  I see this all over the places .. good. !

Reply:
Yeah, would be great to update the most of guidelines, because not all of it in an actual state